common/scummsys.h

```diff
@@ -135,6 +135,7 @@
 	#include <stddef.h>
 	#include <assert.h>
 	#include <ctype.h>
+	#include <stddef.h>
```

Comment:
This is because there are two functionally equivalent commits to Common code to add this header. The PR branch has 976f5732b1b1ccb30198af44ecbe13f156acb06f which is equivalent, but not identical to 849e29dc8749b3d825bb522335cd40a8cf5ad191 which was included in the master branch from PR #871 ... I could drop the commit from this PR branch using rebase, but I am trying to avoid rewriting the history (unless absolutely necessary) and it would result in unbuildable commits if I drop it. The best option here would be to amend the merge commit to remove the duplication by dropping the 976f5732b1b1ccb30198af44ecbe13f156acb06f change, which I will do shortly.